fetch.bs

```diff
@@ -280,7 +280,10 @@ following <a for=struct>items</a>: [[RESOURCE-TIMING]] [[NAVIGATION-TIMING]]
  <dt><dfn export for="fetch timing info">post-redirect start time</dfn> (default 0)
  <dt><dfn export for="fetch timing info">final service worker start time</dfn> (default 0)
  <dt><dfn export for="fetch timing info">final network-request start time</dfn> (default 0)
+ <dt><dfn export for="fetch timing info">first interim network-response start time</dfn> (default 0)
  <dt><dfn export for="fetch timing info">final network-response start time</dfn> (default 0)
+ <dt><dfn export for="fetch timing info">final network-response headers end time</dfn> (default 0)
+ <dt><dfn export for="fetch timing info">response body start time</dfn> (default 0)
```

Comment:
This should have the "final network-" prefix.

Reply:
Done